release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -291,6 +291,32 @@ Currently, it is not supported to use Active Directory Federation Services with
 
 For the full list of OIDC providers, see xref:../authentication/identity_providers/configuring-oidc-identity-provider.adoc#identity-provider-oidc-supported_configuring-oidc-identity-provider[Supported OIDC providers].
 
+[id="ocp-4-11-auth-pod-security-admission"]
+==== Pod security admission
+
+link:https://kubernetes.io/docs/concepts/security/pod-security-admission/[Pod security admission] is now enabled on {product-title}. 
+
+Pod admission is enforced by both pod security and security context constraint (SCC) admissions. Pod security admission runs globally with "privileged" enforcement and "restricted" audit logging and API warnings.
+
+A controller monitors the xref:../authentication/managing-security-context-constraints.adoc[SCC-related permissions] of service accounts in user-created namespaces and automatically labels these namespaces with link:https://kubernetes.io/docs/concepts/security/pod-security-standards/[pod security admission] `warn` and `audit` labels.
```

Comment:
We need to also clarify that the PSA does not work on the namespace prefixed with openshift 
Also, add the info in the release notes that the PSA provides a feature to be enabled and disabled. 
Therefore the list of namespaces that the PSA cannot work at all. 
Also, we ought to make clear that now the "openshift" namespaces are Previleged but we plan to enforce them to "Restricted" in the future so that OCP users can plan better their changes and start to avoid using them to install solutions. 

Therefore, I'd like to suggest we add **something like:** 

```suggestion
A controller monitors the xref:../authentication/managing-security-context-constraints.adoc[SCC-related permissions] of service accounts in user-created namespaces and automatically labels these namespaces with link:https://kubernetes.io/docs/concepts/security/pod-security-standards/[pod security admission] `warn` and `audit` labels.

The SCC synchronization is disabled by default for user-created namespaces that are named with the prefix openshift-. Namespaces that have this prefix are typically system namespaces; by convention, a controller exists to manage them.

This new component also introduces a feature which allows you to enable SCC synchronization in namespaces that have the prefix openshift- by setting the value of the `security.openshift.io/scc.podSecurityLabelSync` label to true. 

However, some namespaces are restricted and do not allow SCC synchronization. These namespaces are system namespaces. You can verify the [list](https://github.com/openshift/cluster-policy-controller/blob/release-4.11/pkg/psalabelsyncer/podsecurity_label_sync_controller.go#L42-L111) of namespaces those is not allowed the synchronization. In this release, these ones are by default enforced with "Privileged".  However, it is planned for the next release to change the performance of these to "Restricted". 


--

```

c/c @stlaz

Reply:
The namespaces prefixed with "openshift" are NOT system one (a least up 4.10). Then, if we want to define it as a rule now it is a new change defined and another point that must be highlighted in the release notes. See that so far we have users and also OLM catalog using these to install Operators. 

Example "openshift-operators" is a default namespace provided in OCP to install operators with the OLM console.

Reply:
I'm going to assume that by PSa you mean the label syncing. 

> the PSA does not work on the namespace prefixed with openshift by default

Keep the release note brief, point to the docs, where we're already documenting this, I believe

> Also, add the info in the release notes that the PSA provides a feature to be enabled and disabled.

Indeed, we might want to have a sentence that this works on an opt-in/opt-out basis that is decribed further in the docs

> make clear that for 4.11 the prefixed "openshift" namespaces have no enforcement for restricted but it is planned to make the restricted on the future release

Good point, if it's not here yet, we should state that our intention for an upcoming release is to be enforcing the "restricted" profile by default and having the automatic labelsyncing work with enforcement labels.